banyand/discovery/discovery.go

```diff
@@ -33,6 +33,7 @@ type ServiceRepo interface {
 	run.Unit
 	bus.Subscriber
 	bus.Publisher
+	GracefulStop()
```

Comment:
Use `run.Service` in place of `GracefulStop`. The lifecycle framework will stop the repo gracefully.

Reply:
Add `stopCh` to ServiceRepo as https://github.com/apache/skywalking-banyandb/blob/main/banyand/queue/local.go#L48 did.

Reply:
ok